source/types/browser.d.ts

```diff
@@ -0,0 +1,338 @@
+// This type was built by cherry picking some types from @types/chrome
+
+declare namespace browser.events {
```

Comment:
I've converted the types that were in the Chrome extension where functions had callbacks. They are now Promises in the types to map to the `browser` object properly.

Reply:
Where did you get this? It’s  probably better to install one of these solutions: https://github.com/mozilla/webextension-polyfill/issues/78, I don’t want us to have to maintain 300 lines of typings 😅

Reply:
Haha, I built it by hand by pulling some out of the `@types/chrome` types. Definitely don't want to maintain something like this, but I wasn't aware of https://github.com/mozilla/webextension-polyfill/issues/78. Thanks for the find. I'll make the change.

Reply:
Fixed. 😅